projects_using_cobra.md

```diff
@@ -6,7 +6,9 @@
 - [Docker (distribution)](https://github.com/docker/distribution)
 - [Etcd](https://etcd.io/)
 - [Gardener](https://github.com/gardener/gardenctl)
+- [GitHub Labeler](https://github.com/erdaltsksn/gh-label)
```

Comment:
This should be after "Github CLI" no?

Reply:
I changed the labeler -> Labeler (Uppercase) after sort the entries. This caused the problem.